platform/integrations/slack_integration/setup.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: setup
+title: Setup
+sidebar_label: Setup
+description: Learn how to install and setup Tolgee Slack app. First, you need to install the app from the Slack App Directory by clicking on the Add to Slack button.
+---
+
+import { FigmaScreenshotWrapper } from '../../shared/_FigmaScreenshotWrapper';
+
```

Comment:
We should tell the user that slack is available in advanced subscription plans and link to the pricing. Like here: https://tolgee.io/platform/projects_and_organizations/members#granular-permissions